tools/measurement/apksize/sdks_ignore.csv

```diff
@@ -0,0 +1,5 @@
+firebase-database-collection
```

Comment:
Why ignore these?

Reply:
@vkryachko can confirm, but I have the impression that these are not actual sdks, but more like helper code or backend, so I wouldn't like to include those in the report.

Reply:
It would still be beneficial to know their sizes, don't you think?

Reply:
Done. PTAL.